internal/xds/translator/testdata/out/extension-xds-ir/http-route-extension-listener-error.listeners.yaml

```diff
@@ -21,18 +21,11 @@
         mergeSlashes: true
         normalizePath: true
         pathWithEscapedSlashesAction: UNESCAPE_AND_REDIRECT
-        routeConfig:
-          name: error_route_configuration
-          virtualHosts:
-          - domains:
-            - '*'
-            name: error_vhost
-            routes:
-            - directResponse:
-                status: 500
-              match:
-                prefix: /
-              name: error_route
+        rds:
+          configSource:
+            ads: {}
+            resourceApiVersion: V3
+          routeConfigName: extension-post-xdslistener-hook-error
```

Comment:
I don’t think we need this test, since the failed xDS wouldn’t be sent to the Envoy fleet anyway.
Would it make more sense to test the xDS output when `failOpen=true` instead?

Reply:
Changed the original test to run in failOpen mode, and duplicated it to also assert errors are retruned in the fail closed option without checking translation results.